code/game/gamemodes/meteor/meteors.dm

```diff
@@ -178,6 +178,7 @@ GLOBAL_LIST_INIT(meteorsC, list(/obj/effect/meteor/dust)) //for space dust event
 		qdel(src)
 
 /obj/effect/meteor/ex_act()
+	SHOULD_CALL_PARENT(FALSE)
```

Comment:
This is not calling parent.
The solution to this is adding an atom-level flag checked at the base of the proc, early returning.